main.js

```diff
@@ -0,0 +1,60 @@
+async function getPlayersData() {
+    var data = await fetch('https://my.api.mockaroo.com/players.json?key=a2166ce0');
```

Comment:
wyciągnąłbym 'https://my.api.mockaroo.com/players.json?key=a2166ce0' to constanta `API_URL`